### ISSUE TITLE
Resolve imports relatively to app.yaml

### ISSUE THREAD

**Issue by zolotov** [go-lang-plugin-org/go-lang-idea-plugin#1525]
Actual for GAE project. Details here: https://cloud.google.com/appengine/docs/go/#Go_Organizing_Go_apps

**Comment by jimbojetlag** [comment]
Sorry to bump on this issue after a month. Is the progress for this issue blocked by other issues?

Working on GAE projects is not productive without resolving this problem.

**Comment by zolotov** [comment]
> Is the progress for this issue blocked by other issues?

Sort of, we have ~130 open issues.

**Comment by jimbojetlag** [comment]
I see. What I meant was if this issue is _technically_ blocked by another issue :) I understand that the workload is high.

**Comment by zolotov** [comment]
>  What I meant was if this issue is technically blocked by another issue :)

No, it's not, just long queue of issue. I'll try to find time for it in a week
